src/types/services/app-versions-service.ts

```diff
@@ -1,7 +1,7 @@
 import { z } from 'zod';
 
-import { appVersionSchema, appVersionsSchema, listAppVersionsSchema } from 'services/schemas/app-versions-schemas';
+import { appVersionHttpSchema, appVersionSchema, listAppVersionsSchema } from 'services/schemas/app-versions-schemas';
 
 export type AppVersion = z.infer<typeof appVersionSchema>;
-export type Version = z.infer<typeof appVersionsSchema>;
+export type Version = z.infer<typeof appVersionHttpSchema>;
 export type ListAppVersionsResponse = z.infer<typeof listAppVersionsSchema>;
```

Comment:
line 7 is a good example- the schema is of the http response, and itself holds the schema of the entity. I would refactor like this:
```suggestion
export type GetAppVersionResponse = z.infer<typeof getAppVersionSchema>;
export type ListAppVersionsResponse = z.infer<typeof listAppVersionsSchema>;
```